### PR TITLE
Update config.example.json

### PR DESCRIPTION
**Pull request by SpeedyCraftah**
I believe merging this PR would vastly benefit the project.

### CODE REVIEW

File: config.example.json [hunk 1]
```diff
@@ -1,10 +1,11 @@
 {
     "token": "",
     "messages": [
-        "brr",
-        "what heck",
-        "beep boop",
-        "pain",
-        "gaming"
+        "nom is cringe",
+        "elian is cringe",
```
Comment (Elian0213) [inline config.example.json:5]: I dont want to spread false information. 

Elian is not cringe.

Rest seems accurate

**Review by Elian0213 — CHANGES_REQUESTED** [review]

File: config.example.json (hunk 1 above)
Comment (SpeedyCraftah) [reply config.example.json:5]: NO

Comment (SpeedyCraftah) [reply config.example.json:5]: actually yes i didnt read sorry

Comment (Elian0213) [reply config.example.json:5]: Please update config.example.json to with requested changes

**Review by SpeedyCraftah — COMMENTED** [review]
Hi. I have amended the file to fit your requested changes,

**Comment by Faith1sGay** [comment]
👀

**Review by Elian0213 — APPROVED** [review]

**Pull request merged**